### PR TITLE
DOC-2381: add new `Math` Plugin documentation for TinyMCE 7.1.

### PR DESCRIPTION
**Pull request by kemister85**
Ticket: DOC-2381

Site: [Staging branch](http://docs-feature-71-doc-2381.staging.tiny.cloud/docs/tinymce/latest/math/)

Changes:
* add new `Math` plugin documentation for TinyMCE 7.1.

Pre-checks:
- [x] Branch prefixed with `feature/<version>/`, `hotfix/<version>/`, `staging/<version>/`, or `release/<version>/`
- [x] `modules/ROOT/nav.adoc` has been updated `(if applicable)`
- [x] Files has been included where required `(if applicable)`
- [x] Files added for `New product features`, and included a `release note` entry.

Review:
- [x] Documentation Team Lead has reviewed

### CODE REVIEW

File: modules/ROOT/examples/live-demos/math/index.js [hunk 1]
```diff
@@ -0,0 +1,9 @@
+tinymce.init({
+  selector: "textarea#math_demo",
+  plugins: [
+      "math", "advlist", "anchor", "autolink", "charmap", "code", "codesample", "fullscreen",
+      "help", "image", "insertdatetime", "link", "lists", "media",
+      "preview", "searchreplace", "table", "visualblocks",
+  ],
+  toolbar: "math | undo redo | styles | bold italic underline strikethrough | alignleft aligncenter alignright alignjustify | bullist numlist outdent indent | link image",
```
Comment (kemister85) [inline modules/ROOT/examples/live-demos/math/index.js:8]:
```suggestion
  toolbar: "math | codesample | undo redo | styles | bold italic underline strikethrough | alignleft aligncenter alignright alignjustify | bullist numlist outdent indent | link image",
```

File: modules/ROOT/partials/index-pages/premium-plugins.adoc [hunk 2]
```diff
@@ -105,6 +105,12 @@ xref:linkchecker.adoc[Link Checker]
 
 Validate links, as they are typed.
 
+a|
+[.lead]
+xref:math.adoc[Math]
+
+Enable insertion of Katex and MathML.
```
Comment (spocke) [inline modules/ROOT/partials/index-pages/premium-plugins.adoc:112]: I think we should use LaTeX instead. We use the KaTeX library for that.

```suggestion
Enable insertion of LaTeX and MathML.
```

File: modules/ROOT/pages/math.adoc [hunk 3]
```diff
@@ -0,0 +1,64 @@
+= Math plugin
+:navtitle: Math Plugin
+:description: {pluginname} plugin provides a way for users to insert a `+<math>+` element into the {productname} editor.
```
Comment (spocke) [inline modules/ROOT/pages/math.adoc:3]: I wonder if we should not mention that it inserts a math element since technically it inserts a `tiny-math-block` or a `tiny-math-inline` element with a `math` element inside at the moment. But future versions might skip the math element though options if you want to render that your self.

```suggestion
:description: {pluginname} plugin provides a way for users to insert math equation into the {productname} editor.
```

File: modules/ROOT/pages/math.adoc [hunk 4]
```diff
@@ -0,0 +1,64 @@
...
+:keywords: plugin, math, formulas
+:pluginname: Math
+:plugincode: math
+:plugincategory: premium
+
+include::partial$misc/admon-paid-addon-pricing.adoc[]
+
+include::partial$misc/admon-requires-7.1v.adoc[]
+
+The {pluginname} plugin provides a way for users to insert complex formulas into `+<math>+` element blocks.
```
Comment (spocke) [inline modules/ROOT/pages/math.adoc:14]:
```suggestion
The {pluginname} plugin provides a way for users to insert complex formulas.
```

File: modules/ROOT/pages/math.adoc [hunk 5]
```diff
@@ -0,0 +1,64 @@
...
+
+To work with math elements, follow these steps:
+
+. Open the {productname} editor with the `math` plugin enabled.
+.. Click the math plugin icon in the toolbar to `insert` or `edit` a math element.
+.. This opens the math dialog which allows the user to choose between either `LaTex` and `MathML` equations for the formulas field.
+.. By default, `LaTeX` is selected.
+. In the math dialog, use the textarea when inserting formulas.
+.. Choose to `insert` the formula either as a **block** paragraph or **inline** with text.
+. Once the formula has been added, the preview `iframe` section within the modal dialog will render the final math equation.
```
Comment (spocke) [inline modules/ROOT/pages/math.adoc:47]: Drop iframe since might not use a iframe in the future more of an implementation detail.

```suggestion
. Once the formula has been added, the preview section within the modal dialog will render the final math equation.
```

File: modules/ROOT/pages/math.adoc [hunk 6]
```diff
@@ -0,0 +1,64 @@
...
+. After reviewing the formula, click the `save` button.
+.. This inserts the finalized math equation into the editor, wrapped in either a `+<div>+` or `+<span>+` element based on the `text-wrap` chosen. The formula is inserted at the last cursor position before the modal dialog was opened.
```
Comment (spocke) [inline modules/ROOT/pages/math.adoc:49]:
```suggestion
.. This inserts the finalized math equation into the editor, wrapped in either a `+<tiny-math-block>+` or `+<tiny-math-inline>+` element based on the `text-wrap` chosen. The formula is inserted at the last cursor position before the modal dialog was opened.
```

File: modules/ROOT/pages/math.adoc [hunk 7]
```diff
@@ -0,0 +1,64 @@
...
+
+[NOTE]
+{pluginname} uses the below `custom_elements` to insert complex mathematical content into {productname}. {companyname} does not recommend using custom_element names for `blockMathTag` and `inlineMathTag` to avoid issues when using the {pluginname}.
+
+Learn more about Custom elements, see xref:content-filtering.adoc#custom_elements[custom_elements] and xref:content-filtering.adoc#example-using-custom_elements-with-presets-such-as-blocks-phrasing-and-flow[example using custom_elements].
```
Comment (spocke) [inline modules/ROOT/pages/math.adoc:54]: I think we can skip this it uses custom elements for wrapping the math formulas but not sure we need to mention that again it's mentioned above.

**Review by spocke — CHANGES_REQUESTED** [review]

File: modules/ROOT/examples/live-demos/math/index.html [hunk 8]
```diff
@@ -0,0 +1,63 @@
+<textarea id="math_demo">
+  <p style="text-align: center;">The following equations are represented in the HTML source code as LaTex/MathML expressions.</p>
```
Comment (kemister85) [inline modules/ROOT/examples/live-demos/math/index.html:2]:
```suggestion
  <p style="text-align: center;">The following equations are represented in the HTML source code as LaTeX/MathML expressions.</p>
```

File: modules/ROOT/partials/misc/admon-requires-7.1v.adoc [hunk 9]
```diff
@@ -0,0 +1 @@
+NOTE: This feature is only available for {productname} 7.1.0 and later.
```
Comment (FarzadHayat) [inline modules/ROOT/partials/misc/admon-requires-7.1v.adoc:1]: Looking at other required version admonitions, typically we don't include the patch version (the `.0`) unless the new feature was released in a patch version.

Comment (FarzadHayat) [inline modules/ROOT/partials/misc/admon-requires-7.1v.adoc:1]:
```suggestion
NOTE: This feature is only available for {productname} 7.1.0 and later.
```
```suggestion
NOTE: This feature is only available for {productname} 7.1 and later.
```

Comment (FarzadHayat) [reply modules/ROOT/partials/misc/admon-requires-7.1v.adoc:1]: Looking at other required version admonitions, typically we don't include the patch version (the .0) unless the new feature was released in a patch version.

File: modules/ROOT/examples/live-demos/math/index.html [hunk 10]
```diff
@@ -0,0 +1,63 @@
+<textarea id="math_demo">
+  <p style="text-align: center;">The following equations are represented in the HTML source code as LaTeX/MathML expressions.</p>
+
+  <div style="text-align: center;">
+	<h1>LaTex Example</h1>
+	<aside style="background-color: #f9f9f9; border-left: 5px solid #ccc; padding: 10px; margin: 10px 0; font-style: italic; color: #555;">
+	  The following example is a representation of a LaTex expression after it was inserted as a block element after selecting "Text above and below" from the text wrap toggle.
```
Comment (FarzadHayat) [inline modules/ROOT/examples/live-demos/math/index.html:7]:
```suggestion
	<h1>LaTeX Example</h1>
	<aside style="background-color: #f9f9f9; border-left: 5px solid #ccc; padding: 10px; margin: 10px 0; font-style: italic; color: #555;">
	  The following example is a representation of a LaTeX expression after it was inserted as a block element after selecting "Text above and below" from the text wrap toggle.
```

File: modules/ROOT/pages/math.adoc [hunk 11]
```diff
@@ -0,0 +1,129 @@
...
+});
+----
+
+== How to Use
+
+To work with math elements, follow these steps:
+
+. Open the {productname} editor with the {pluginname} plugin enabled.
+.. Click the {pluginname} plugin icon in the toolbar to `insert` or `edit` a math element.
+.. This opens the {pluginname} dialog which allows the user to choose between either `LaTex` and `MathML` equations for the formulas field.
```
Comment (FarzadHayat) [inline modules/ROOT/pages/math.adoc:43]:
```suggestion
.. This opens the {pluginname} dialog which allows the user to choose between either `LaTeX` and `MathML` equations for the formulas field.
```

File: modules/ROOT/pages/math.adoc [hunk 12]
```diff
@@ -0,0 +1,129 @@
...
+. Once the formula has been added, the preview section within the modal dialog will render the final math equation.
+. After reviewing the formula, click the `save` button.
+.. This inserts the math equation into the editor, wrapped in either a `+<tiny-math-block>+` or `+<tiny-math-inline>+` element based on the **Text Wrap** chosen.
+
+[TIP]
+The formula is inserted at the last cursor position before the modal dialog was opened.
+
+=== Inserting LaTeX formulas (Block)
+
+The <tiny-math-block formula=""> custom tag is used to wrap the <math>, <semantics>, and <mrow> tags when inserting a LaTeX formulas when "Text above and below" `Text Wrap` is selected.
```
Comment (FarzadHayat) [inline modules/ROOT/pages/math.adoc:56]:
```suggestion
The `+<tiny-math-block formula="">+` custom tag is used to wrap the `+<math>+`, `+<semantics>+`, and `+<mrow>+` tags when inserting a LaTeX formulas, when the `Text Wrap` is set to `Text above and below`.
```

File: modules/ROOT/pages/math.adoc [hunk 13]
```diff
@@ -0,0 +1,129 @@
...
+      <mrow>
+      </mrow>
+    </semantics>
+  </math>
+</tiny-math-block>
+----
+
+=== Inserting LaTeX formulas (Inline)
+
+The <tiny-math-inline formula=""> custom tag is used to wrap the <math>, <semantics>, and <mrow> tags when inserting a LaTeX formulas, when the `Text Wrap` is set to `Inline with Text`.
```
Comment (FarzadHayat) [inline modules/ROOT/pages/math.adoc:72]:
```suggestion
The `+<tiny-math-inline formula="">+` custom tag is used to wrap the `+<math>+`, `+<semantics>+`, and `+<mrow>+` tags when inserting a LaTeX formulas, when the `Text Wrap` is set to `Inline with Text`.
```

File: modules/ROOT/pages/math.adoc [hunk 14]
```diff
@@ -0,0 +1,129 @@
...
+      <mrow>
+      </mrow>
+    </semantics>
+  </math>
+</tiny-math-inline></p>
+----
+
+=== Inserting MathML formulas (Block)
+
+The <tiny-math-block> custom tag is used to wrap the <math>, <mrow>, and <mn> tags when inserting a MathML formulas when "Text above and below" `Text Wrap` is selected.
```
Comment (FarzadHayat) [inline modules/ROOT/pages/math.adoc:88]:
```suggestion
The `+<tiny-math-block>+` custom tag is used to wrap the `+<math>+`, `+<mrow>+`, and `+<mn>+` tags when inserting a MathML formulas, when the `Text Wrap` is set to `Text above and below`.
```

File: modules/ROOT/pages/math.adoc [hunk 15]
```diff
@@ -0,0 +1,129 @@
...
+     <mn>
+     </mn>
+    </mrow>
+  </math>
+</tiny-math-block>
+----
+
+=== Inserting MathML formulas (Inline)
+
+The <tiny-math-block-inline> custom tag is used to wrap the <math>, <mrow>, and <mn> tags when inserting a MathML formulas when the `Text Wrap` is set to `Inline with Text`.
```
Comment (FarzadHayat) [inline modules/ROOT/pages/math.adoc:104]:
```suggestion
The `+<tiny-math-block-inline>+` custom tag is used to wrap the `+<math>+`, `+<mrow>+`, and `+<mn>+` tags when inserting a MathML formulas when the `Text Wrap` is set to `Inline with Text`.
```

File: modules/ROOT/partials/menu-item-ids/math-menu-items.adoc [hunk 16]
```diff
@@ -0,0 +1,5 @@
+[cols="1,1,2",options="header"]
+|===
+|Menu item identifier |xref:menus-configuration-options.adoc#example-the-tinymce-default-menu-items[Default Menu Location] |Description
+|`+math+` |Insert |Provides a way for users to insert a `+<math>+` element into the {productname}.
```
Comment (FarzadHayat) [inline modules/ROOT/partials/menu-item-ids/math-menu-items.adoc:4]:
```suggestion
|`+math+` |Insert |Provides a way for users to insert a `+<math>+` element into {productname}.
```

File: modules/ROOT/partials/commands/math-cmds.adoc [hunk 17]
```diff
@@ -0,0 +1,11 @@
+[cols="1,3",options="header"]
+|===
+|Command |Description
+|Math |This command opens the {pluginname} dialog.
```
Comment (FarzadHayat) [inline modules/ROOT/partials/commands/math-cmds.adoc:4]:
```suggestion
|Math |This command opens the Math dialog.
```

Comment (FarzadHayat) [reply modules/ROOT/partials/commands/math-cmds.adoc:4]: editor-command-identifiers.adoc uses this partial and does not know what `{pluginname}` is.

**Review by FarzadHayat — APPROVED** [review]

File: modules/ROOT/examples/live-demos/math/index.html [hunk 18]
```diff
@@ -0,0 +1,63 @@
...
+
+	<p style="text-align: center;">To understand this expression, let's consider what happens as <code>\(n\)</code> increases:</p>
+
+	<ul style="text-align: center;">
+	  <li>When <code>\(n\)</code> is large, the term <code>\(\frac{1}{n}\)</code> tends to zero, which implies that <code>\((1 + \frac{1}{n})\)</code> approaches 1.</li>
+	  <li>If <code>\(N\)</code> represents infinity, the limit is also 1, because the additional term <code>\(\frac{1}{n}\)</code> becomes negligible.</li>
+	  <li>If <code>\(N\)</code> is a finite value, the limit would be <code>\((1 + \frac{1}{N})\)</code>. This reflects the behavior as <code>\(n\)</code> approaches a specific finite number.</li>
+	</ul>
+
+	<p style="text-align: center;">Therefore, without further context about what <code>\(N\)</code> represents, the specific result of the limit depends on whether <code>\(N\)</code> is infinite or finite.</p>
```
Comment (FarzadHayat) [inline modules/ROOT/examples/live-demos/math/index.html:61]: Is this section necessary? seems irrelevant to the plugin functionality.

Comment (kemister85) [reply modules/ROOT/examples/live-demos/math/index.html:61]: The content in this demo, will be replaced with an updated version as requested by product.

File: modules/ROOT/partials/commands/math-cmds.adoc (hunk 17 above)
Comment (kemister85) [reply modules/ROOT/partials/commands/math-cmds.adoc:4]: Screenshot 2024-05-02 at 6 05 13 am
@FarzadHayat if a variable is set in the header of the parent `.adoc` file such as the plugin.adoc file, any partials rendered inside the document will render as expected.

File: modules/ROOT/partials/toolbar-button-ids/math-toolbar-buttons.adoc [hunk 19]
```diff
@@ -0,0 +1,5 @@
+[cols="1,3",options="header"]
+|===
+|Toolbar button identifier |Description
+|`+Math+` |Opens the math modal dialog.
```
Comment (ltrouton) [inline modules/ROOT/partials/toolbar-button-ids/math-toolbar-buttons.adoc:4]:
```suggestion
|`+math+` |Opens the math modal dialog.
```

File: modules/ROOT/partials/menu-item-ids/math-menu-items.adoc [hunk 20]
```diff
@@ -0,0 +1,5 @@
+[cols="1,1,2",options="header"]
+|===
+|Menu item identifier |xref:menus-configuration-options.adoc#example-the-tinymce-default-menu-items[Default Menu Location] |Description
+|`+math+` |Insert |Provides a way for users to insert a `+<math>+` element into {productname}.
```
Comment (ltrouton) [inline modules/ROOT/partials/menu-item-ids/math-menu-items.adoc:4]: The description should be consistent with the toolbar button description. I think a basic description is fine
```suggestion
|`+math+` |Insert |Opens the math modal dialog.
```

File: modules/ROOT/pages/math.adoc [hunk 21]
```diff
@@ -0,0 +1,129 @@
+= Math plugin
+:navtitle: Math Plugin
+:description: {pluginname} plugin provides a way for users to insert math equation into the {productname} editor.
```
Comment (ltrouton) [inline modules/ROOT/pages/math.adoc:3]:
```suggestion
:description: {pluginname} plugin allows users to insert math equations into the {productname} editor.
```

File: modules/ROOT/pages/math.adoc [hunk 22]
```diff
@@ -0,0 +1,129 @@
...
+
+The {pluginname} plugin provides a way for users to insert complex formulas.
+
+== Interactive example
+
+liveDemo::{plugincode}[]
+
+== Basic setup
+
+To setup the {pluginname} plugin user-interface in the editor:
```
Comment (ltrouton) [inline modules/ROOT/pages/math.adoc:22]:
```suggestion
To set up the {pluginname} plugin user interface in the editor:
```

File: modules/ROOT/pages/math.adoc [hunk 23]
```diff
@@ -0,0 +1,129 @@
...
+. In the {pluginname} dialog, use the textarea when inserting formulas.
+.. Choose to `insert` the formula either as a **block** paragraph or **inline** with text.
+. Once the formula has been added, the preview section within the modal dialog will render the final math equation.
+. After reviewing the formula, click the `save` button.
+.. This inserts the math equation into the editor, wrapped in either a `+<tiny-math-block>+` or `+<tiny-math-inline>+` element based on the **Text Wrap** chosen.
+
+[TIP]
+The formula is inserted at the last cursor position before the modal dialog was opened.
+
+=== Inserting LaTeX formulas (Block)
```
Comment (ltrouton) [inline modules/ROOT/pages/math.adoc:54]: Do we need to have separate headings for both Latex and MathML or can we just have one math block heading and one math inline heading? They result in essentially the same HTML except for the `formula` attribute.

**Review by ltrouton — APPROVED** [review]

File: modules/ROOT/examples/live-demos/math/index.html [hunk 24]
```diff
@@ -0,0 +1,63 @@
...
+  </div>
+</textarea>
```
Comment (ShiridiGandham) [inline modules/ROOT/examples/live-demos/math/index.html:63]:
```suggestion
<textarea id="math_demo">
<h2>Computation of Definite Integrals.</h2>
		<p>In computing definite integrals it is important to observe that when the integral is known to exist the limit can be calculated on any properly chosen subset of the&nbsp;<tiny-math-inline formula="S_\delta"></tiny-math-inline>'s. So we&nbsp;have that if <tiny-math-inline formula="S_{\delta_1}"></tiny-math-inline>,&nbsp;<tiny-math-inline formula="S_{\delta_2}"></tiny-math-inline>, ... is any sequence of sums such that&nbsp;<tiny-math-inline formula="\displaystyle\mathop{L}_{n\doteq\infty}\delta_n=0"></tiny-math-inline>, then</p>
		<tiny-math-block formula="\mathop{L}_{n\doteq\infty} S_{\delta_n} = \int_a^b f(x)dx."></tiny-math-block>
		<p>One case of this kind occurs when&nbsp;<tiny-math-inline formula="\xi_k"></tiny-math-inline>&nbsp;is taken as an end-point of the interval <tiny-math-inline formula="{x_{k-1}}{x_k}"></tiny-math-inline> and all the&nbsp;<tiny-math-inline formula="\Delta_kx"></tiny-math-inline>'s are equal. Then we have</p>
		<tiny-math-block formula="\displaystyle \int_a^b f(x)dx =  \mathop{L}_{n\doteq\infty} \sum_{k=1}^n f(a+k\Delta x)\Delta x,  \text{ where }  \Delta x=\frac{b-a}{n}."></tiny-math-block>
		<p>A simple example of this principle is the proof of the following&nbsp;theorem.</p>
		<p><em>Exerpt from <a href="https://www.gutenberg.org/ebooks/18741">Introduction to Infinitesimal Analysis; Functions of One Real Variable by Veblen et al.</a></em></p>	
	</textarea>
```

File: modules/ROOT/pages/math.adoc [hunk 25]
```diff
@@ -0,0 +1,129 @@
...
+----
+<tiny-math-block-inline>
+  <math>
+    <mrow>
+     <mn>
+     </mn>
+    </mrow>
+  </math>
+</tiny-math-block-inline>
+----
```
Comment (ShiridiGandham) [inline modules/ROOT/pages/math.adoc:116]: +

File: modules/ROOT/pages/math.adoc [hunk 26]
```diff
@@ -0,0 +1,129 @@
...
+
```
Comment (ShiridiGandham) [inline modules/ROOT/pages/math.adoc:117]:
```suggestion

```

File: modules/ROOT/pages/math.adoc [hunk 27]
```diff
@@ -0,0 +1,129 @@
...
+. Open the {productname} editor with the {pluginname} plugin enabled.
+.. Click the {pluginname} plugin icon in the toolbar to `insert` or `edit` a math element.
+.. This opens the {pluginname} dialog which allows the user to choose between either `LaTeX` and `MathML` equations for the formulas field.
+.. By default, `LaTeX` is selected.
+. In the {pluginname} dialog, use the textarea when inserting formulas.
+.. Choose to `insert` the formula either as a **block** paragraph or **inline** with text.
+. Once the formula has been added, the preview section within the modal dialog will render the final math equation.
+. After reviewing the formula, click the `save` button.
+.. This inserts the math equation into the editor, wrapped in either a `+<tiny-math-block>+` or `+<tiny-math-inline>+` element based on the **Text Wrap** chosen.
```
Comment (ShiridiGandham) [inline modules/ROOT/pages/math.adoc:49]:
```suggestion
. Enable Math Plugin
+
.. Open the {productname} editor with the {pluginname} plugin enabled.
+
. Insert or Edit {pluginname} Elements
.. Click the {pluginname}  plugin icon in the toolbar.
.. This opens the {pluginname} dialog.
.. Select preferred **language**
.. Enter {pluginname} **formula** in the textarea.
.. Choose preferred **text wrap**
+
. Preview and Save
.. Check {pluginname} formula output in **preview.**
.. Review the final {pluginname} equation in the **preview** section.
.. Click the **save** button in the {pluginname} dialog.
.. The {pluginname} equation is inserted into the editor, wrapped accordingly based on the chosen **text wrap.**
```

File: modules/ROOT/pages/math.adoc [hunk 28]
```diff
@@ -0,0 +1,79 @@
...
+.. Check {pluginname} formula output in **preview.**
+.. Review the final {pluginname} equation in the **preview** section.
+.. Click the **save** button in the {pluginname} dialog.
+.. The {pluginname} equation is inserted into the editor, wrapped accordingly based on the chosen **text wrap.**
++
+. **Formula Tag Wrapping**
+.. The {pluginname} dialog wraps the inserted formula with a specific tag.
+.. The choice of tag depends on the selected **text wrap** option in the dialog:
+... For block (Text above and below) text wrap, the tag used is <tiny-math-block>.
+... For inline (Inline with text) text wrap, the tag used is <tiny-math-inline>.
```
Comment (ShiridiGandham) [inline modules/ROOT/pages/math.adoc:62]:
```suggestion
**Formula Tag Wrapping**
. The {pluginname} dialog wraps the inserted formula with a specific tag.
. The choice of tag depends on the selected **text wrap** option in the dialog:
.. For block (Text above and below) text wrap, the tag used is <tiny-math-block>.
.. For inline (Inline with text) text wrap, the tag used is <tiny-math-inline>.
```

File: modules/ROOT/pages/math.adoc [hunk 29]
```diff
@@ -0,0 +1,79 @@
...
+.. Check {pluginname} formula output in **preview.**
+.. Review the final {pluginname} equation in the **preview** section.
+.. Click the **save** button in the {pluginname} dialog.
+.. The {pluginname} equation is inserted into the editor, wrapped accordingly based on the chosen **text wrap.**
++
+**Formula Tag Wrapping**
+. The {pluginname} dialog wraps the inserted formula with a specific tag.
+. The choice of tag depends on the selected **text wrap** option in the dialog:
+.. For block (Text above and below) text wrap, the tag used is <tiny-math-block>.
+.. For inline (Inline with text) text wrap, the tag used is <tiny-math-inline>.
```
Comment (ShiridiGandham) [inline modules/ROOT/pages/math.adoc:62]:
```suggestion
+

//

**Formula Tag Wrapping**

. The {pluginname} dialog wraps the inserted formula with a specific tag.
. The choice of tag depends on the selected **text wrap** option in the dialog:
.. For block (Text above and below) text wrap, the tag used is <tiny-math-block>.
.. For inline (Inline with text) text wrap, the tag used is <tiny-math-inline>.
```

File: modules/ROOT/pages/math.adoc [hunk 30]
```diff
@@ -0,0 +1,83 @@
...
++
+
+//
+
+**Formula Tag Wrapping**
+
+. The {pluginname} dialog wraps the inserted formula with a specific tag.
+. The choice of tag depends on the selected **text wrap** option in the dialog:
+.. For block (Text above and below) text wrap, the tag used is <tiny-math-block>.
+.. For inline (Inline with text) text wrap, the tag used is <tiny-math-inline>.
```
Comment (ShiridiGandham) [inline modules/ROOT/pages/math.adoc:66]:
```suggestion
.. For block (_Text above and below_) text wrap, the tag used is <tiny-math-block>.
.. For inline (_Inline with text_) text wrap, the tag used is <tiny-math-inline>.
```

File: modules/ROOT/pages/math.adoc [hunk 31]
```diff
@@ -0,0 +1,83 @@
...
+.. Choose preferred **text wrap**
++
+. **Preview and Save**
+.. Check {pluginname} formula output in **preview.**
+.. Review the final {pluginname} equation in the **preview** section.
+.. Click the **save** button in the {pluginname} dialog.
+.. The {pluginname} equation is inserted into the editor, wrapped accordingly based on the chosen **text wrap.**
++
+
+//
```
Comment (tiny-ben-tran) [inline modules/ROOT/pages/math.adoc:59]:
```suggestion
```

File: modules/ROOT/partials/index-pages/premium-plugins.adoc [hunk 32]
```diff
@@ -105,6 +105,12 @@ xref:linkchecker.adoc[Link Checker]
 
 Validate links, as they are typed.
 
+a|
+[.lead]
+xref:math.adoc[Math]
+
+Enable insertion of LaTeX and MathML.
```
Comment (tiny-ben-tran) [inline modules/ROOT/partials/index-pages/premium-plugins.adoc:112]:
```suggestion
Enable insertion of math formulas in Latex and MathML markups.
```
??

**Review by tiny-ben-tran — APPROVED** [review]

File: modules/ROOT/pages/math.adoc [hunk 33]
```diff
@@ -0,0 +1,83 @@
...
+
+**Formula Tag Wrapping**
+
+. The {pluginname} dialog wraps the inserted formula with a specific tag.
+. The choice of tag depends on the selected **text wrap** option in the dialog:
+.. For block (_Text above and below_) text wrap, the tag used is <tiny-math-block>.
+.. For inline (_Inline with text_) text wrap, the tag used is <tiny-math-inline>.
```
Comment (TheSpyder) [inline modules/ROOT/pages/math.adoc:66]:
```suggestion
.. For block (_Text above and below_) text wrap, the tag used is `<tiny-math-block>`.
.. For inline (_Inline with text_) text wrap, the tag used is `<tiny-math-inline>`.
```

File: modules/ROOT/pages/math.adoc [hunk 34]
```diff
@@ -0,0 +1,83 @@
...
+
+
+[TIP]
+The formula is inserted at the last cursor position before the modal dialog was opened.
```
Comment (TheSpyder) [inline modules/ROOT/pages/math.adoc:70]: All dialogs that insert content do this, I don't think we need to state it (nor do we for any other plugin dialog).
```suggestion
```

File: modules/ROOT/partials/index-pages/premium-plugins.adoc (hunk 32 above)
Comment (TheSpyder) [reply modules/ROOT/partials/index-pages/premium-plugins.adoc:112]: How about
```suggestion
Enable insertion of math formulas using Latex or MathML markup.
```

**Review by TheSpyder — APPROVED** [review]

File: modules/ROOT/pages/math.adoc [hunk 35]
```diff
@@ -0,0 +1,83 @@
...
+== How to Use
+
+To work with math elements, follow these steps:
+
+. **Enable Math Plugin**
++
+.. Open the {productname} editor with the {pluginname} plugin enabled.
++
+. **Insert or Edit {pluginname} Elements**
+.. Click the {pluginname}  plugin icon in the toolbar.
```
Comment (spocke) [inline modules/ROOT/pages/math.adoc:46]: I'm guessing spaces will be removed by the browser rendering.

```suggestion
.. Click the {pluginname} plugin icon in the toolbar.
```

**Review by spocke — APPROVED** [review]
